streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -86,16 +86,18 @@ public void process(final K key, final V value) {
      * @return a string representation of this node, useful for debugging.
      */
     public String toString() {
-        StringBuilder sb = new StringBuilder("");
-        sb.append(name + ": ");
+        StringBuilder sb = new StringBuilder(name);
+        sb.append(":");
         if (stateStores != null && !stateStores.isEmpty()) {
-            sb.append("stateStores [");
-            for (String store : (Set<String>) stateStores) {
-                sb.append(store + ",");
+            sb.append("\n\t\t\t\t\tstates:\t\t[");
```

Comment:
I thought about it before, but it is a bit tricky to do since it could be dynamic based on which class the `toString` function is triggered first.  Let me think about it more and see if I can come with a clean solution.